composer.json

```diff
@@ -32,7 +32,13 @@
         "wunderio/drupal-ping": "^1.0"
     },
     "require-dev": {
-        "webflo/drupal-core-require-dev": "~8.5"
+        "codeception/codeception": "^2.5",
```

Comment:
Having all these dependencies in the project rather than the CI environments makes a lot of sense.